python/setup.py

```diff
@@ -6,7 +6,7 @@ def has_ext_modules(foo):
 
 setup(
     name='visenze_recordio',
-    version='0.1',
+    version='0.2',
```

Comment:
Where does this version number come from? I checked [gemfury](https://manage.fury.io/dashboard/visenze/package/pkg_21uYbX/version/ver_Ux1VK) and it was still 0.1.

Reply:
after this one is merged then it'll be released to gemfury as 0.2

Reply:
I see.